### PR TITLE
many: teach prepare-image to copy the model assertion (and prereqs) into the seed area of the image

### PR DESCRIPTION
**Pull request by pedronis**
This teaches prepare-image to copy the model assertion and its prereqs (obtained from the store) into the seed area of the image.

### CODE REVIEW

**Comment by pedronis** [comment]
based on #1702

**Comment by pedronis** [comment]
rebase on #1713

File: tests/main/prepare-image-grub/task.yaml [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
+    STORE_ADDR: localhost:11028
+prepare: |
+    . $TESTSLIB/store.sh
+    setup_store fake $STORE_DIR
+restore: |
+    . $TESTSLIB/store.sh
+    teardown_store fake $STORE_DIR
     rm -rf $ROOT
 execute: |
-    echo Creating model assertion
```
Comment (mvo5) [inline tests/main/prepare-image-grub/task.yaml]: Really really nice!

**Comment by mvo5** [comment]
:+1: Might be nice to extract some bits in bootstrapToRootDir() into a helper, but that can be done later, I really like it!

**Comment by chipaca** [comment]
:+2: 😝

**Pull request merged**